boat_nav/src/path_planners/planner_base.py

```diff
@@ -181,29 +183,16 @@ def publish_target(*argv):
 	@staticmethod
 	def _calc_wind_coming():
 		"""Calculate the angle of the wind."""
-		Planner.new_wind_heading = (Planner.ane_reading + Planner.cur_boat_heading) % 360
-		Planner.wind_coming = (Planner.new_wind_heading + 180) % 360
+		Planner.new_wind_heading = angles.normalize(Planner.ane_reading + Planner.cur_boat_heading)
```

Comment:
There's a calc_wind_apparent utility so why not a wind_coming utility? Or should they both just live here as static methods? I feel like they should be grouped together wherever they are.

Reply:
Not exactly sure what `calc_wind_apparent` you're talking about, though I did a quick search and found an awful lot of nodes that subscribe to `/anemometer` only to convert it to apparent wind heading, which makes sense since the raw anemometer reading isn't usually that helpful. I propose we have the apparent wind heading calculated in a central node (`sensor_parser_node` probably), and then publish the resulting `/apparent_wind_heading`.

Fewer nodes need wind_coming, but I'm happy to let them calculate it themselves in the `/apparent_wind_heading` callback, especially with `boat_utilities.angles.opposite()`.

Let's move this discussion to a new issue since it's beyond the scope of this PR.

Reply:
You're right I was mistaken about the `calc_wind_apparent` that was just a function in one of the nodes.  Yeah with the opposite function I feel good about this.